pkg/term/console_windows.go

```diff
@@ -20,21 +27,255 @@ const (
 	// If parameter is a screen buffer handle, additional values
 	ENABLE_PROCESSED_OUTPUT   = 0x0001
 	ENABLE_WRAP_AT_EOL_OUTPUT = 0x0002
+
+	//http://msdn.microsoft.com/en-us/library/windows/desktop/ms682088(v=vs.85).aspx#_win32_character_attributes
+	FOREGROUND_BLUE       = 1
+	FOREGROUND_GREEN      = 2
+	FOREGROUND_RED        = 4
+	FOREGROUND_INTENSITY  = 8
+	FOREGROUND_MASK_SET   = 0x000F
+	FOREGROUND_MASK_UNSET = 0xFFF0
```

Comment:
@sachin-jayant-joshi I don't want to export all these from `pkg/term`. Maybe we could do `pkg/term/ansi` for ansi-specific things, and `pkg/term/console` for windows-specific things? If @tianon is okay we could import it with `import . "github.com/.../pkg/term/internal/console"` eliminating the need for a verbose `console.` prefix.